docs/admin-guide/migration/in-place/index.md

```diff
@@ -0,0 +1,186 @@
+        sudo apt-add-repository 'deb http://repos.azulsystems.com/ubuntu stable main' -y
+        sudo apt-get update
+        sudo apt-get install zulu-8 -y
+
+
+### Application Server
+
+!!! todo
+
+    I would prefer if we install Tomcat under /opt and then create a symbolic link /opt/tomcat to 
+    the version you want to use. 
+
+=== "Tomcat 9 (APT)"
+
+        sudo apt install tomcat9 
+
+=== "Tomcat 9 (Manual)"
+
+        wget https://archive.apache.org/dist/tomcat/tomcat-9/v9.0.91/bin/apache-tomcat-9.0.91.tar.gz
+        tar xfv apache-tomcat-9.0.91.tar.gz
+        sudo mv apache-tomcat-9.0.91 /opt/tomcat9
+        sudo chown -R tomcat:tomcat /opt/tomcat9
+        sudo chmod +x /opt/tomcat9/bin/*
+        rm apache-tomcat-9.0.91.tar.gz
+
+=== "Tomcat 8.5 (Manual)"
+
+        wget https://archive.apache.org/dist/tomcat/tomcat-8/v8.5.89/bin/apache-tomcat-8.5.89.tar.gz
+        tar xfv apache-tomcat-8.5.89.tar.gz
+        sudo mv apache-tomcat-8.5.89 /opt/tomcat85
+        sudo chown -R tomcat:tomcat /opt/tomcat85
+        sudo chmod +x /opt/tomcat85/bin/*
+        rm apache-tomcat-8.5.89.tar.gz
+
+### Web Server / HTTPS / SSL (optional)
+This step is optional since you can stay with Apache 2, if that's what was installed previously. We
+have started to migrate some of our production environments to use nginx, so that's why these
+instructions are included.
+
+=== "Apache"
+
+    If you've configured SSL/HTTPS on Apache 2 then you can stay with that configuration. You might 
+    need to re-configure Apache 2 to delegate requests to Tomcat 8.5 or Tomcat 9 through mod_jk or 
+    whatever plugin was used. We'll try to upgrade the docs to include those instructions soon.
+    
+    !!! todo
+    
+            Include configuration changes for mod_jk or mod_http or whatever. 
+
+=== "nginx"
+    
+    #### Install nginx apt
+    
+        sudo apt-get install nginx -y
+    
+    #### Configure SSL
+    
+        sudo touch /etc/nginx/conf.d/default.conf
+        cat 9_nginx_config.template > /etc/nginx/conf.d/default.conf
+        echo "Bruteforce creating missing files in the /etc/letsencrypt"
+        curl https://raw.githubusercontent.com/certbot/certbot/master/certbot-nginx/certbot_nginx/_internal/tls_configs/options-ssl-nginx.config > /etc/letsencrypt/options-ssl-nginx.conf
+        curl https://raw.githubusercontent.com/certbot/certbot/master/certbot/certbot/ssl-dhparams.pem > /etc/letsencrypt/ssl-dhparams.pem
+    
+    #### Disable Apache2 
+    
+        service apache2 stop
+        systemctl daemon-reload
+        systemctl disable apache2
+    
+    #### Enable Nginx
+    
+        systemctl enable nginx
+        service nginx start
+
+=== "Tomcat"
+    
+    This documentation does not officially support enabling HTTPS/SSL on Tomcat, but there's 
+    nothing preventing you from using this feature.  
+
+## Upgrade Application
+
+### Backup Database 
+
+### Configuration
```

Comment:
In 0.8.x we were using both `openboxes-config.properties` and `openboxes-config.groovy` and for 0.9.x we switched to having only `openboxes-config.groovy` on PIH instances of OB, but since both still should work as before I am not sure if there is anything that needs to be mentioned here about that.

Reply:
i am advocating for openboxes.yml for new installs (https://docs.openboxes.com/en/obpih-6197/admin-guide/installation/ubuntu2204/configuration/), but there's nothing explicitly stated for parallel and in-place migrations. for the migrations, we should probably state that config should be copied as-is onto the new server / new directory structure.